pkg/acsengine/defaults-kubelet.go

```diff
@@ -136,6 +144,11 @@ func setKubeletConfig(cs *api.ContainerService) {
 		}
 		setMissingKubeletValues(profile.KubernetesConfig, o.KubernetesConfig.KubeletConfig)
 
+		if profile.OSType == "Windows" {
+			// Remove Linux-specific values
+			delete(profile.KubernetesConfig.KubeletConfig, "--pod-manifest-path")
```

Comment:
Ha. I wonder if we should do this for Linux as well. Instead of this recent change that @tariq1890 made:

https://github.com/Azure/acs-engine/pull/3744/files